streams/src/test/java/org/apache/kafka/streams/integration/KTableKTableForeignKeyJoinMaterializationIntegrationTest.java

```diff
@@ -94,15 +101,49 @@ public void before() {
         );
     }
 
+    @Test
+    public void shouldEmitRecordWhenJoiningWithDefaultSerdes() {
```

Comment:
Hey @bellemare , do we need to add a new test? It looks like this condition was intended to be covered already in KTableKTableForeignKeyJoinScenarioTest . Should we just fix that test instead?

Reply:
Okay, will check.

Reply:
@vvcephei Just getting back to this now. Before I rework KTableKTableForeignKeyJoinScenarioTest:

I can keep it simple and try something like:
KTable<String, Long> aTable
KTable<Long, Integer> bTable

Or I can go ahead and use the JSONSerde, User, and Foo classes. I think the former will suffice for coverage purposes, but I like the latter because it may better reflect real-life use-cases (and I usually look to the tests myself to show me how to use a function). Thoughts on this before I refactor?

Reply:
Went with "keep it simple" basic Integer and String.